Extension/src/Utility/Sandbox/interfaces.ts

```diff
@@ -0,0 +1,53 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import { is } from '../System/guards';
+
+export interface ScriptError {
+    line: number;
+    column: number;
+    message: string;
+    file: string;
+    category: number;
+    code: number;
+    offset: number;
+}
+
+export interface CreateOptions {
+    /** the "filename" of the source to compile */
+    filename?: string;
+
+    /** the column in the physical file where the source starts
+     *
+     * @default 0
+    */
+    columnOffset?: number;
+
+    /** the line in the physical file where the source starts
+     *
+     * @default 0
+     */
+    lineOffset?: number;
+
+    /**
+     * set to true to invoke the TS transpiler (to resolve imports/exports/etc)

```

Comment:
Should these 2 TS references be TypeScript? I'm not familiar with TS is something different or not.

Reply:
removing...

Reply:
removed reference to it.